spec/system/cookies_consent_spec.rb

```diff
@@ -0,0 +1,157 @@
+require "rails_helper"
+
+describe "Cookies consent" do
+  before do
+    Setting["feature.cookies_consent"] = true
```

Comment:
I know it's the default option but, just so the test is easier to read, what do you think about adding:

```ruby
    Setting["cookies_consent.version_name"] = "v1"
```